cosmo_tester/test_suites/test_simple_manager_blueprint/abstract_single_host_test.py

```diff
@@ -72,8 +72,10 @@ def setup_simple_manager_env(self):
             self.local_env.outputs()['simple_vm_private_ip_address']
 
     def bootstrap_simple_manager_blueprint(self, override_inputs=None):
-        self.manager_blueprints_repo_dir = clone(MANAGER_BLUEPRINTS_REPO_URL,
-                                                 self.workdir)
+        self.manager_blueprints_repo_dir = clone(
+            MANAGER_BLUEPRINTS_REPO_URL, self.workdir,
+            branch=os.environ.get('CLOUDIFY_MANAGER_BLUEPRINTS_BRANCH',
```

Comment:
This functionality isn't actually that useful. I need to get a specific commit from the master repo, but if I set BRANCH_NAME_CORE, it tries to use that for everything. Either way, I'll work around that by hard coding it locally.